include/mesh/checkpoint_io.h

```diff
@@ -35,6 +35,15 @@ namespace libMesh
 {
 // Forward declarations
 class Xdr;
+class CheckpointIO;
+
+/**
+ * splitmesh takes the given initialized/opened mesh and partitions it into nsplits pieces or
+ * chunks.  It returns a CheckpointIO object that can be used to write the mesh chunks into
+ * individual files (e.g. by calling checkpoint_obj.write(out_file_name)).  This function supports
+ * MPI parallelism and can be used with several MPI procs to speed up splitting.
+ */
```

Comment:
This comment should probably make the "N procs, M files" case support or lack thereof clearer.